crates/agglayer-node/src/kernel/mod.rs

```diff
@@ -75,9 +75,11 @@ impl<RpcProvider> Kernel<RpcProvider> {
             .get(&rollup_id)
             .ok_or(ZkevmNodeVerificationError::InvalidRollupId(rollup_id))?;
 
-        Ok(ZkevmNodeClient::new(
-            jsonrpsee::http_client::HttpClientBuilder::new().build(url.as_str())?,
-        ))
+        let client = jsonrpsee::http_client::HttpClientBuilder::new()
+            .request_timeout(std::time::Duration::from_secs(45))
```

Comment:
How did you end up choosing 45 secs?
This shouldn't be a fixed value but a configurable one using the config file

Reply:
> How did you end up choosing 45 secs?

As written in the PR description:

> If the request to the agglayer node is dropped before that timeout, we do not get any useful diagnostic. So lower it to 45s as well, so we increase chances that our request to the L2 node times out before the client request to agglayer, which gives us more room to log the error. The requests to L2 nodes are rather trivial so 45s should be more than enough. This is only a partial fix / workaround, ideally cancelled requests are logged regardless of timeout.

So it's basically the common default of 60s minus some slack for various latencies.